### ISSUE TITLE
Unresponsive after changing scene with pick object node

### ISSUE THREAD

**Issue by Maluveh** [armory3d/armory#1326]
Thanks for helping to contribute to Armory3D!
To help us fix your issue or implement your feature as quickly as possible, please fill out the following information as best you can:

### SHORT DESCRIPTION:
_Keep it 1-3 sentences and to the point._

### EXPECTED RESULT:
_What did you expect to happen? For a feature request, what would you like to happen?_

### ACTUAL RESULT:
_What happens right now that's different from how you expect or want it to work?_

### STEPS TO REPRODUCE:
_Please list clear, step by step instructions to replicate the result you got. For a feature request list the steps to use the feature._
Steps:
1.
2.
3.
etc.

### TECHNICAL SPECIFICATIONS:
_What version(s) of the following did you try:_
Blender:
Armory:
Operating system:
Graphics card & driver:

### TEST FILE TO REPRODUCE:
_Please create the simplest file possible that demonstrates the issue:_
*(drag & drop the zipped .blend file here)*

**Comment by Maluveh** [comment]
I built a menu which makes use of the pick object node. When I hit the how to play button. It moves to the how to play scene but when i want to return to the menu the scene will be unresponsive. The blend file is attached. 

https://drive.google.com/file/d/10BDxCy6wNSqC8oy3-N5f3XQUdalfAM2T/view

**Comment by zaethan** [comment]
tl;dr:
Try to use "Started" instead of "Down" for the mouse nodes that you use for your buttons.

long version:
One thing I noticed is that you are using the mouse node with "Down" for clicking the buttons (except in the return to menu trait).
This means that as long as the mouse button is pressed, the node will activate.

I guess this is what is happening (I may be wrong though):
You are hitting the how to play button and are being brought to the how to play scene. Then you try to exit the scene by clicking again, without moving your mouse. This will bring you back to your title screen.
Since you will (most likely) hold the mouse button down for more than one frame, the how to play button activates again (because "down" activates every frame, as long as the mouse is clicked), bringing you right back to the how to play screen, looking as if nothing happened (being unresponsive).

After changing all button related traits to use "started" it seems to work fine here.

**Comment by Maluveh** [comment]
Thank you
